www/src/pages/index.js

```diff
@@ -84,7 +84,7 @@ const ButtonToolbar = styled('div')`
     margin: -1rem;
 
     & > * {
-      width: 100%;
+      width: calc(100% - 1.85rem);
```

Comment:
Should actually be `calc(100% - 2rem)` since the margins added on each side sum up to 2.